mathewDavidov/assignments/twopointers/lc1/lc1.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Problem Statement: Given an array of sorted numbers and a target sum, find a
+ * pair in the array whose sum is equal to the given target. Write a function to
+ * return the indices of the two numbers (i.e. the pair) such that they add up
+ * to the given target.
+ * 
+ * Edge cases: 
+ * empty array / null array -> empty array 
+ * no target pair found -> empty array
+ * 
+ * Left pointer will begin at the first index (0) 
+ * Right pointer will begin at the last index (nums.length - 1) 
+ * while left < right 
+ *   create a currentSum equal to the numbers at the indices left and right 
```

Comment:
Using words like "declare" and "initialize" will show your interviewer that you can use the technical jargon.